app/src/main/java/com/sdp/movemeet/models/Image.java

```diff
@@ -0,0 +1,75 @@
+package com.sdp.movemeet.models;
+
+import android.net.Uri;
+import android.widget.ImageView;
+
+/**
+ *
+ * This class represents a message
+ *
+ */
+public class Image implements FirebaseObject {
+    private Uri imageUri;
+    private ImageView imageImageView;
+    private String documentPath;
+
+    /**
+     * Constructor for a new image
+     *
+     * @param imageUri URI pointing to the image file
+     */
+    public Image(Uri imageUri, ImageView imageImageView) {
+
+        if (imageUri == null && imageImageView == null) {
```

Comment:
Should be || instead of &&

Reply:
No, this is in fact intended. In some situation, the `imageUri` can be `null` (for cases in which we **fetch** the image from Firebase Storage, such as when we retrieve the user profile picture in `ProfileActivity` and `EditProfileActivity`, or the header picture in `ActivityDescriptionActivity` or the chat image in `MessageViewHolder` from Firebase Storage, because we just have the path of the image in Firebase Storage, but not yet its URI) and in some other situations, the `imageImageView` can be `null` as well (for cases in which we **upload** the image to Firebase Storage, such as in `ChatActivity` for instance where we specify the URI of the image to upload to Firebase Storage, but not its ImageView (which is useless at this point)), but they can't be both simultaneously `null`! That's the reason why I use here a `&&` and not a `||`symbol.